### PR TITLE
Custom 404 page 🦕

### PR DESCRIPTION
**Pull request by meszarosdezso**
Today I realised that deno.land doesn't have a custom 404 page, so I decided to create one.

Here's a demo:

https://deno-website2-35x36av75.vercel.app/fourohfour

deno404page

### CODE REVIEW

File: components/NotFound.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      <Header />
+      <header>
+        <h1 className="font-extrabold text-5xl leading-10 tracking-tight text-gray-900">
+          404
+        </h1>
+        <h2 className="mt-4 sm:mt-5 font-light text-2xl text-center leading-tight text-gray-900">
+          <strong className="font-semibold">Deno</strong> and{" "}
+          <strong className="font-semibold">Ferris</strong> couldn't find what
+          you're looking for.
+        </h2>
```
Comment (ry) [inline components/NotFound.tsx:16]: Can you replace this h2 text. Either with "not found" or removed entirely.  The "ferris" bit will confuse people not steeped in programming language lore.

**Review by ry — COMMENTED** [review]
Awesome artwork! Thank you!

**Comment by crowlKats** [comment]
I think it should have the footer as well

**Comment by meszarosdezso** [comment]
@crowlKats @ry I could make it so that the footer is the land itself, what do you think?

**Comment by meszarosdezso** [comment]
Screenshot 2020-10-23 at 21 22 16

**Comment by ry** [comment]
that looks good - thanks

**Comment by meszarosdezso** [comment]
Oops, it still needs an `overflow-x: hidden`. Working on it.

**Review by ry — APPROVED** [review]
Thanks

**Pull request merged**